datocms.json

```diff
@@ -322,7 +322,7 @@
       },
       "migration": {
         "name": "Migration",
-        "value": "No"
+        "value": "Yes"
```

Comment:
https://www.datocms.com/docs/scripting-migrations/scripting-migrations-with-the-datocms-cli